composer.json

```diff
@@ -3,16 +3,16 @@
     "description": "Bundle for creating forms in Sulu.",
     "type": "sulu-bundle",
     "require": {
-        "symfony/symfony": "^2.8.7 || ^3.0",
-        "symfony/swiftmailer-bundle": "^2.3 || ^3.0",
+        "symfony/symfony": "^2.8.7 || ^3.0 || ^4.0",
```

Comment:
we should not longer require  the whole `symfony/symfony` framework here we should replace this with:

```
        "symfony/config": "^3.4 || ^4.0",
        "symfony/dependency-injection": "^3.4 || ^4.0",
        "symfony/event-dispatcher": "^3.4 || ^4.0",
        "symfony/form": "^3.4 || ^4.0",
        "symfony/validator": "^3.4 || ^4.0",
```